docker/back/api/nginx/vhost.conf

```
@@ -0,0 +1,25 @@
+server_tokens off;
+
```

```suggestion
map $http_x_request_id $request_uid {
    default   $http_x_request_id;
    ""        $request_id;
}
```
Permet de définir un ID de requête unique qui suit une requete et ces sous requêtes